Policies/github-corporate-terms-of-service.md

```diff
@@ -62,7 +62,7 @@ You must create an individual account before you can create a Corporate Account.
 - You must be a human to create an account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts:
   - A machine account is an account set up by an individual human who accepts the Terms on behalf of the account, provides a valid email address, and is responsible for its actions. A machine account is used exclusively for performing automated tasks. Multiple users may direct the actions of a machine account, but the owner of the account is ultimately responsible for the machine's actions. You may maintain no more than one free machine account in addition to your free personal account.
 - One person or legal entity may maintain no more than one free account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).
-- You may not create an account for the use of any User under the age of 13. While we are thrilled to see brilliant young coders get excited by learning to program, we must comply with United States law. GitHub does not target our Service to children under 13, and we do not permit any Users under 13 on our Service. If we learn of any User under the age of 13, we will [terminate that User’s account immediately](#l-cancellation-and-termination). If you are a resident of a country outside the United States, your country’s minimum age may be older; in such a case, you are responsible for complying with your country’s laws.
+- User of a corporate account must be either older than 13 (people under this age will be referred to as an "underage" in the following text), or consented by a person's account who is over 24 who is the parent or the legal guardian of the said minor (referred to as a "consenter"). Consenters are required to verify that he/she/it is said underage's parent or legal guardian. GitHub may employ any kind of software checks and/or request identity information in the verification requests to verify said fact. Once consented, that consent cannot be revoked even if any of those accounts are closed in the future. This is required because underages cannot consent by themselves in the most of the world, including the United States, in which GitHub has its legal personhood. Violation of this policy leads to [termination and deletion of all of your code and all their forks](#l-cancellation-and-termination). If you are a resident of a country outside the United States, your country’s minimum age may be older; in such a case, you are responsible for complying with your country’s laws. Age restrictions does not apply to legal entities creating corporate accounts but do apply to their representatives if exist.
```

Comment:
`he/she/it` maybe instead use 'he/she/they' since `it` might just mean that the account is a robot, and we want to include people who do not prefer to use the `he/she` pronouns.